.github/workflows/build.yml

```diff
@@ -72,6 +72,16 @@ jobs:
       - name: Test
         run: npm run test:ci
 
+      - name: Upload Dist Files
+        uses: actions/upload-artifact@v3
+        with:
+          name: dist
+          path: |
+            apps/wing/dist
+            apps/wing/*.wasm
+            libs/wingsdk/lib
+            libs/wingsdk/.jsii
```

Comment:
These files are already updated in the next steps, (some are technically in tgz files)